lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -33,7 +34,13 @@ def sync(mediator)
         @containers.each(&:reset_status)
       end
 
-      display_logs(mediator) if unmanaged? && deploy_succeeded?
+      if unmanaged?
```

Comment:
I think this block is ok for now. But I think a future refactor should have the resource watcher be responsible for triggering the logging.